src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const arr = Object.values(robot);
+
+  for (let i = 0; i < arr.length; i++) {
+    for (let k = i + 1; k <= arr.length; k++) {
+      if (arr[i] === arr[k]) {
+        return null;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    robot[robot[key]] = key;
+    delete robot[key];
+  }
+
+  return robot;
 }
```

Comment:
you need only one for in loop here(and hasOwnProperty for checking)

Reply:
@marina-tilniak , I have been thinking about it whole week and still have no idea how to solve with one one for in loop and hasOwnProperty, need some hint )

Reply:
it is done )